gensim/models/doc2vec_corpusfile.pyx

```diff
@@ -153,7 +153,8 @@ def d2v_train_epoch_dbow(model, corpus_file, offset, start_doctag, _cython_vocab
 
     cdef int i, j, document_len
     cdef int effective_words = 0
-    cdef int total_effective_words = 0, total_documents = 0, total_words = 0
+    cdef int total_effective_words = 0, total_documents = 0
```

Comment:
How about changing these too, to be on the safe side (especially `total_effective_words`? Any reason not to?

Reply:
There are `long long`s in many other places. Do you want explicit sizes everywhere?

Reply:
I think it makes sense, but is not critical.

More important to me is that any variable that holds a potentially  large count (token frequency, number of tokens in a corpus, number of sentences in a corpus, …) is at least 64 bit.